src/reconstruction/CoreModelCoupled.jl

```diff
@@ -362,40 +362,96 @@ function change_coupling_bounds!(
     end
 end
 
-"""
-    change_bounds!(
-        model::CoreModelCoupled,
-        rxns::Vector{Int};
-        xl::V = Float64[],
-        xu::V = Float64[],
-    )
+@doc @_change_bound_s_bang("CoreModelCoupled", "rxn_idxs", "Vector{Int}", "[2, 43]", true, true)
+function change_bounds!(
+    model::CoreModelCoupled,
+    reaction_idxs::Vector{Int};
+    lower_bounds = fill(-_constants.default_reaction_bound, length(rxns)),
+    upper_bounds = fill(_constants.default_reaction_bound, length(rxns)),
+)   
+    for (rxn_idx, lb, ub) in zip(reaction_idxs, lower_bounds, upper_bounds)
+        change_bound!(model, rxn_idx; lower_bound=lb, upper_bound=ub)
+    end
+end
 
-Change the lower and/or upper bounds ('xl' and 'xu') for given reactions.
+@doc @_change_bound_s_bang("CoreModelCoupled", "rxn_idx", "Int", "2", false, true)
+function change_bound!(
+    model::CoreModelCoupled,
+    rxn::Int;
+    lower_bound = -_constants.default_reaction_bound,
+    upper_bound = _constants.default_reaction_bound,
+)
+    model.lm.xl[rxn] = lower_bound    
+    model.lm.xu[rxn] = upper_bound
+    return nothing # so that nothing gets printed
+end
 
-"""
+@doc @_change_bound_s_bang("CoreModelCoupled", "rxn_ids", "Vector{String}", "[\"PFL\", \"FBA\"]", true, true)
 function change_bounds!(
+    model::CoreModelCoupled,
```

Comment:
use `args...; kwargs...`

Reply:
won't work here I think, the arguments get changed slightly